awx/api/serializers.py

```diff
@@ -3515,6 +3519,7 @@ class Meta:
 
     def get_related(self, obj):
         res = super(WorkflowJobSerializer, self).get_related(obj)
+        res.pop('execution_environment', None)  # EEs aren't meaningful for workflows
```

Comment:
It should be impossible to set this anyway, but sure.

Reply:
I want to say that is so, but for workflow jobs intermediate ones might get it anyway due to being triggered within tasks.py.